AlfrescoSDK/AlfrescoSDK/Constants/AlfrescoInternalConstants.h

```diff
@@ -387,3 +387,12 @@ extern NSString * const kAlfrescoPersonPropertyCompanyEmail;
 extern NSString * const kAlfrescoDefaultBackgroundNetworkSessionId;
 extern NSString * const kAlfrescoDefaultBackgroundNetworkSessionSharedContainerId;
 
+/**---------------------------------------------------------------------------------------
+ * @name Cloud Connection Status
+ --------------------------------------------------------------------------------------- */
+typedef NS_ENUM(NSInteger, CloudConnectionStatus)
```

Comment:
Enum types and members need to be prefixed with "Alfresco" to avoid linker problems with third party apps.

Actually, this is something I missed on the initial diagnostic feature. The `ConnectionDiagnosticStatus` enum should also be prefixed with "Alfresco" - especially so in this case as it's a public constant.